src/app/pages/home/home.component.html

```diff
@@ -6,22 +6,23 @@
       <div class="card-head p-4">
         <h1 class="text-center title-y">Elegí un avatar y escribí tu nombre</h1>
       </div>
-      <div class="card-body p-4">
+      <div class="card-body p-4 justify-content-center">
+        <app-avatar-image [images]="avatarImages" (avatarSelected)="selectAvatar($event)"></app-avatar-image>
         <form class="card-footer flex-fill">
-            <app-avatar-image [images]="avatarImages"></app-avatar-image>
           <input
             type="text"
             class="bg-purpel"
-            placeholder="Escribí tu nombre" />
+            placeholder="Escribí tu nombre" 
+            (input)="onNameInputChange($event)"/>
         </form>
       </div>
     </div>
-    <div class="d-flex flex-row justify-content-center gap-4 p-4">
-      <app-custom-button>Iniciar partida</app-custom-button>
+    <div class="d-flex flex-row justify-content-center gap-4">
+      <app-custom-button (click)="iniciarPartida()">Iniciar partida</app-custom-button>
```

Comment:
The button text does not describe the action it actually performs, change it to "Crear sala".